cobra/core/model.py

```diff
@@ -778,6 +778,43 @@ def to_array_based_model(self, deepcopy_model=False, **kwargs):
         from cobra.core.arraybasedmodel import ArrayBasedModel
         return ArrayBasedModel(self, deepcopy_model=deepcopy_model, **kwargs)
 
+    def slim_optimize(self, error_value=0., message=None):
```

Comment:
Yeah that work. Only if I want the return value to be `None` if not optimal, there is no way now, right?